### PR TITLE
Russian language support

### PR DESCRIPTION
**Pull request by TomahawkNSK**
Added translation into Russian language

### CODE REVIEW

File: CQUI.modinfo [hunk 1]
```diff
@@ -264,11 +264,16 @@
         <LocalizedText id="CQUI_TEXT">
             <Items>
                 <File>Assets/Text/Gossip_Text.xml</File>
+                <File>Assets/Text/Gossip_Text_de.xml</File>
```
Comment (alexeyOnGitHub) [inline CQUI.modinfo:267]: What's up with these new "it" and de file references?

**Review by alexeyOnGitHub — CHANGES_REQUESTED** [review]

File: CQUI.modinfo (hunk 1 above)
Comment (the-m4a) [reply CQUI.modinfo:267]: it appears those files exist, (Gossip_Text_de.xml and Gossip_Text_it.xml) but aren't currently referenced by the modinfo?

There is an error in the Gossip_Text_de.xml that is easily fixed: 
Line 4 has one too many dashes, between "Text" and "GERMAN", it's confusing for XML parsers
```xml
		<!-- Replace Gossip Text -- GERMAN -->
```
Change that to

**Review by the-m4a — APPROVED** [review]
Spot checked a few with "translate to english" and they looked fine... good catch on the DE and IT files that are currently not referenced by the modinfo file, and very much appreciate the effort!

File: CQUI.modinfo (hunk 1 above)
Comment (TomahawkNSK) [reply CQUI.modinfo:267]: Yes, those files exist, but aren't currently referenced by the modinfo.
I also fixed an error in the comment.

File: Assets/Text/Gossip_Text_ru.xml [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+			<Text>[ICON_Razed] {1_CityName} подвергается осаде.</Text>
+		</Replace>
+		<Replace Tag="LOC_GOSSIP_CITY_LIBERATED" Language="ru_RU">
+			<Text>[ICON_Buildings] {1_CivName} освободил {2_CityName} и вернул его {3_CivName}.</Text>
+		</Replace>
+		<Replace Tag="LOC_GOSSIP_CITY_RAZED" Language="ru_RU">
+			<Text>[ICON_Razed] {1_CivName} разрушил город, основанный {2_CivName}.</Text>
+		</Replace>
+		<Replace Tag="LOC_GOSSIP_CLEAR_CAMP" Language="ru_RU">
+			<Text>[ICON_Barbarian] {1_CivName} использовал {LOC_GRAMMAR_A_AN &lt;&lt; {2_UnitName}} чтобы очистить {LOC_GRAMMAR_A_AN &lt;&lt; {3_ImprovementName}}.</Text>
```
Comment (alexeyOnGitHub) [inline Assets/Text/Gossip_Text_ru.xml:40]: should have a comma before "chtobi"

File: Assets/Text/Gossip_Text_ru.xml [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+			<Text>[ICON_Declare_formal_war] {1_CivName} начал военное наступление на {2_CivName}.</Text>
+		</Replace>
+		<Replace Tag="LOC_GOSSIP_WAR_PREPARATION" Language="ru_RU">
+			<Text>[ICON_Declare_formal_war] {1_CivName} готовится к войне с {2_CivName}.</Text>
+		</Replace>
+		<Replace Tag="LOC_GOSSIP_INQUISITION_LAUNCHED" Language="ru_RU">
+			<Text>[ICON_Religion] {1_CivName} начал инквизицию против {2_ReligionName}.</Text>
+		</Replace>
+		<Replace Tag="LOC_GOSSIP_LAND_UNIT_LEVEL" Language="ru_RU">
+			<Text>[ICON_Promotion] {1_CivName} имеет {2_UnitName}, который заработал {3_PromotionName} повышение.</Text>
```
Comment (alexeyOnGitHub) [inline Assets/Text/Gossip_Text_ru.xml:97]: I would place "{3_PromotionName}" after "повышение".

File: Assets/Text/Gossip_Text_ru.xml [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+		</Replace>
+		<Replace Tag="LOC_GOSSIP_MAKE_DOW" Language="ru_RU">
+			<Text>[ICON_Declare_formal_war] {1_CivName} объявил войну {2_OtherPlayer}!</Text>
+		</Replace>
+		<Replace Tag="LOC_GOSSIP_NATIONAL_PARK_CREATED" Language="ru_RU">
+			<Text>[ICON_Terrain] {1_CivName} создал {2_NationalParkName}.</Text>
+		</Replace>
+		<Replace Tag="LOC_GOSSIP_NAVAL_UNIT_LEVEL" Language="ru_RU">
+			<Text>[ICON_Promotion] {1_CivName} имеет {2_UnitName}, который заработал {3_PromotionName} повышение.</Text>
```
Comment (alexeyOnGitHub) [inline Assets/Text/Gossip_Text_ru.xml:106]: similarly here - "{3_PromotionName}" at the end.

File: Assets/Text/Gossip_Text_ru.xml [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+		</Replace>
+		<Replace Tag="LOC_GOSSIP_NEW_RELIGIOUS_MAJORITY" Language="ru_RU">
+			<Text>[ICON_Religion] Доминирующая религия в {1_CivName} сейчас: {2_ReligionName}.</Text>
```
Comment (alexeyOnGitHub) [inline Assets/Text/Gossip_Text_ru.xml:109]: maybe "teper" instead of "seichas" given that this is a change in status?

**Review by alexeyOnGitHub — APPROVED** [review]

**Pull request merged**